ctx.py

```diff
@@ -13,20 +13,50 @@
 # limitations under the License.
 
 import functools
+import os
 
+from pydash import get
+import yaml
+
+import util
 '''
 Execution context. Filled upon invocation of cli.py, read by submodules
 '''
 
 args=None # the parsed command line arguments
 
+# TODO: Maybe provide a map name->path ?
+config=None
+
+# Load $HOME/.cc-utils.cfg if it exists
+if 'HOME' in os.environ:
+    util_config_path = os.path.join(os.environ['HOME'], '.cc-utils.cfg')
+    try:
+        with open(util_config_path) as config_file:
+            config = yaml.load(config_file.read())
+    except FileNotFoundError:
+        # could not find file - fail silently
+        pass
+    except Exception as e:
+        util.verbose(str(e))
+
+def get_config(path: str):
```

Comment:
reflect the possible absense in the name (e.g. get_config_or_none); also, according to our naming conventions, getters do not have get prefix ;)

Reply:
also, consider defining a configuration class (which would then offer this getter)